semgrep-core/Analyzing/Makefile

```diff
@@ -6,7 +6,7 @@ TARGET=lib
 OPAMPKG=pfff-lang_GENERIC-analyze
 
 SRC= \
-  Naming_AST.ml Normalize_AST.ml Constant_propagation.ml \
+  Typing.ml Naming_AST.ml Normalize_AST.ml Constant_propagation.ml \
```

Comment:
I think Typing_AST.ml would have been better, to be consistent with some of the other files.